### DISCUSSION TITLE
Automatically add the 'scene condition' to the 'Observer manager'

### DISCUSSION THREAD

**Discussion by DriesDeRidder** [FirstGearGames/FishNet#363] [Documentation]
Hi FGG,

I had an issue where I upgraded from Unity 2021 to Unity 2022. In general everything went great but for some reason - when reinstalling Fishnet - my observer manager lost it's connection to the 'scene condition'. I then had errors in some scenes where I had networked scene objects. 

It didn't happen in all scenes so I was quite lost and spent a week looking for why it happened. I thought the fishnet version was broken, so I downloaded several versions. I rebuild Scene ID's like a 100 times, reimported the library, Downgraded to 2021 again,...

finally I went all the way back to a commit before the version change and went back over all components while comparing the 2 projects and saw that the scene condition was missing. It's a component that I rarely use because once it's set-up, you don't really have to do anything with it anymore.

Anyway, a lot of hassle for such a small fix.

So my request would be that maybe, if no condition is set in the observer manager, that the default is 'scene condition'. 
Since I think that in most cases having no conditions would be not desirable. 

Maybe add a bool that is standard false to 'override default' or something that allows you to have no conditions?

This is a picture of the errors I kept getting:
(I know it says something about the observermanager in the error but since I don't use it often, it didn't really ring a bell)

![image](https://user-images.githubusercontent.com/32333783/236819397-63c8761e-0255-4372-b9f7-0c7886aae1d6.png)

**Comment by FirstGearGames** [comment]
There are reasons why someone may not want the scene condition. I won't be making this change because it could break projects, and trust me, Unity will find another way to break during your updates :P
